src/qml/pages/node/PeerDetails.qml

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import org.bitcoincore.qt 1.0
+import "../../controls"
+import "../../components"
+
+Page {
+    signal backClicked()
+
+    property PeerDetailsModel details
+
+    Connections {
+        target: details
+        function onDisconnected() {
+            root.backClicked()
+        }
+    }
+
+    id: root
+    background: null
+
+    header: NavigationBar2 {
+        leftItem: NavButton {
+            iconSource: "image://images/caret-left"
+            text: qsTr("Back")
+            onClicked: root.backClicked()
+        }
+        centerItem: Header {
+            headerBold: true
+            headerSize: 18
+            header: qsTr("Peer " + details.nodeId)
+        }
+    }
+
+    component PeerKeyValueRow: Row {
+        width: parent.width
+        property string key: ""
+        property string value: ""
+        CoreText {
+            color: Theme.color.neutral9;
+            text: key;
+            width: 125;
+            horizontalAlignment: Qt.AlignLeft;
+        }
+        CoreText {
+            color: Theme.color.neutral9;
+            elide: Text.ElideRight;
+            wrapMode: Text.WordWrap;
+            text: value
+            width: parent.width - 125;
+            horizontalAlignment: Qt.AlignLeft;
+        }
+    }
+
+    ScrollView {
+        id: scrollView
+        width: parent.width
+        height: parent.height
+        clip: true
+        contentWidth: width
+
+        Column {
+            width: Math.min(parent.width - 40, 450)
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 10
+
+            CoreText {
+                text: "Information";
+                bold: true;
+                font.pixelSize: 18;
+                horizontalAlignment: Qt.AlignLeft;
+                color: Theme.color.neutral9;
+            }
+            Column {
+                width: parent.width
+                spacing: 5
```

Comment:
The spacing between sections of information is actually 15, so we'd want to add a bottom margin of 5

```diff

diff --git a/src/qml/pages/node/PeerDetails.qml b/src/qml/pages/node/PeerDetails.qml
index 224aef8c7..b32c2a0c7 100644
--- a/src/qml/pages/node/PeerDetails.qml
+++ b/src/qml/pages/node/PeerDetails.qml
@@ -79,6 +79,7 @@ Page {
             Column {
                 width: parent.width
                 spacing: 5
+                bottomPadding: 5
                 PeerKeyValueRow { key: qsTr("Address"); value: details.address }
                 PeerKeyValueRow { key: qsTr("VIA"); value: details.addressLocal }
                 PeerKeyValueRow { key: qsTr("Type"); value: details.type }
@@ -101,6 +102,7 @@ Page {
             Column {
                 width: parent.width
                 spacing: 5
+                bottomPadding: 5
                 PeerKeyValueRow { key: qsTr("Starting block"); value: details.startingHeight }
                 PeerKeyValueRow { key: qsTr("Synced headers"); value: details.syncedHeaders }
                 PeerKeyValueRow { key: qsTr("Synced blocks"); value: details.syncedBlocks }
@@ -116,6 +118,7 @@ Page {
             Column {
                 width: parent.width
                 spacing: 5
+                bottomPadding: 5
                 PeerKeyValueRow { key: qsTr("Direction"); value: details.direction }
                 PeerKeyValueRow { key: qsTr("Connection time"); value: details.connectionDuration }
                 PeerKeyValueRow { key: qsTr("Last send"); value: details.lastSend }

```

Reply:
See here, this starts to look like the design file
| master | patch |
| ------ | ----- |
| <img width="708" alt="section-spacing-master" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/8ca90790-41b7-4cda-bbb4-8fc132114f8b"> | <img width="752" alt="section-spacing-patch" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/2860b3f1-46e4-494f-a6b8-b22ca3801c21"> |

Design file screenshot of section spacing: 

<img width="1684" alt="spacing-between-sections-15" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/48536952-7e6f-49b7-8252-443f810a3933">